ClemBot.Bot/bot/cogs/trivia_cog.py

```diff
@@ -0,0 +1,566 @@
+            case 0:
+                if input[0].isnumeric():
+                    question_number = int(input[0])
+
+                    if 0 < question_number <= 50:
+                        return question_number
+                else:
+                    raise UserInputError(
+                        "Question Number has to be a number within the range of 1 to 50")
+            case 1:
+                if input[1].isnumeric():
+                    trivia_number = int(input[1])
+                    if 0 < trivia_number <= 24:
+                        return trivia_number + 8
+                    elif trivia_number == 0:
+                        return None
+                    else:
+                        raise UserInputError(
+                            "Category Number out of bounds(Number has to be 1-24) or enter the category you want! Type ?trivia help to see the category list")
+                else:
+                    trivia_category = input[1].lower()
+
+                    for x in CATEGORYLIST_LOWER:
+                        if x.find(trivia_category) != -1:
+                            return_this = CATEGORYLIST_LOWER.index(x) + 9
+                            return return_this
+                    else:
+                        raise UserInputError("Category not found!")
+            case 2:
+                if input[2].isnumeric():
+                    evaluate_int = int(input[2])
+                    if 0 < evaluate_int <= 3:
+                        return_string = DIFFICULTY_LOWER[evaluate_int - 1]
+                        return return_string
+                    elif evaluate_int == 0:
+                        return None
+                    else:
+                        raise UserInputError(
+                            "Difficulty Number out of bounds(Number has to be 1-3) or enter Easy-Hard! Type ?trivia help to see the difficulty list.")
+                else:
+                    difficulty = input[2].lower()
+                    for x in DIFFICULTY_LOWER:
+
+                        if (
+                                x.find(difficulty) != -1):  #Searches the substring. If this ever comes up, its not a bug you can type in a and not find your category GIGO. It is better than exact case parsing.
+                            return x
+                    else:
+                        raise UserInputError("Difficulty not found")
+            case 3:
+                if input[3].isnumeric():
+                    evaluate_int = int(input[3])
+                    if 0 < evaluate_int < 3:
+                        final_return = QUESTIONTYPE[evaluate_int - 1]
+                        return final_return
+                    elif evaluate_int == 0:
+                        return None
+                    else:
+                        raise UserInputError(
+                            "Question type number out of bounds(1 or 2) 1: Multiple Choice 2: Boolean. Type ?trivia help to see our question types.")
```

Comment:
We need to grab the actual command prefix for the context guild and interpolate it into this string